packages/pigeon/pigeons/core_tests.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:pigeon/pigeon.dart';
+
+// A class containing all supported types.
+class AllTypes {
```

Comment:
I originally changed this from `Everything` when I was adding instead of migrating; I could change it back now that there's no collision; I think this is a clearer name, but I don't feel strongly if anyone else does.

Reply:
I think this makes more sense. I'm fine with it.